src/components/button/Button.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from "prop-types";
+import tw from "twin.macro";
+import theme from "@/themes/theme";
+import styled from "styled-components";
+
+const Button = styled.button(
```

Comment:
This is an older version of the Button component.  When merging this component we need to make sure that we use the updated version of the Button component.

Reply:
We should merge that PR first and then rebase this branch